src/components/Combobox/utils/constants.js

```diff
@@ -1,13 +1,23 @@
 import PropTypes from 'prop-types';
 
-
 export const itemType = PropTypes.shape({
+  icon: PropTypes.shape({
+    icon: PropTypes.string.isRequired,
+    sprite: PropTypes.string.isRequired,
+  }),
+  id: PropTypes.string.isRequired,
+  isHeader: PropTypes.bool,
+  label: PropTypes.string.isRequired,
+  meta: PropTypes.node,
+});
+
+export const itemTypeEntity = PropTypes.shape({
```

Comment:
`itemType` and `itemTypeEntity` are absolutely identical. Why do we need both of them?

Reply:
`itemTypeEntity` has a required `icon.isRequired` as per the LDs specs